Documentation/ceph-common-issues.md

```diff
@@ -495,7 +541,10 @@ If your cluster is larger than a few nodes, you may get lucky enough that the mo
 old state. Looking at the OSD pod logs you will see an error about the file already existing.
 
 ```console
-$ kubectl -n rook-ceph logs rook-ceph-osd-fl8fs
```

Comment:
keep as single block since it has the pod name?

Reply:
Alternately, this can be

```console
kubectl -n rook-ceph logs deploy/rook-ceph-osd-ID
```